app/commands.py

```diff
@@ -316,6 +317,14 @@ def insert_inbound_numbers_from_file(file_name):
     file.close()
 
 
+@notify_command(name='replay-create-pdf-letters')
+@click.option('-n', '--notification_id', required=True,
+              help="Notification id of the letter that needs the create_letters_pdf task replayed")
```

Comment:
is it worth having `type=click.UUID` here to do validation?

note, would then need to cast back to a str in the apply_async.

Reply:
done